src/KeyVault/KeyVault.Test/KeyVault.Test.csproj

```diff
@@ -13,11 +13,11 @@
   <ItemGroup>
     <PackageReference Include="Microsoft.Azure.KeyVault" Version="3.0.1" />
     <PackageReference Include="Microsoft.Azure.KeyVault.WebKey" Version="3.0.1" />
-    <PackageReference Include="Microsoft.Azure.Management.KeyVault" Version="4.0.0-preview.1" />
     <PackageReference Include="Microsoft.Azure.Management.Network" Version="24.0.0" />
   </ItemGroup>
 
   <ItemGroup>
+    <ProjectReference Include="..\KeyVault.Sdk\KeyVault.Sdk.csproj" />
```

Comment:
I assume this is not required because SDK.csproj is introduced by KeyVault.csproj. Please help double check.